tibco_ems/tests/conftest.py

```diff
@@ -0,0 +1,20 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield
+
+
+@pytest.fixture
+def instance():
+    return {
```

Comment:
👏 👏 finally someone with common sense. I never understood the need to define the instance in `common.py`, then deepcopy it in `conftest.py` and provide it as a fixture.